subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/JvmProjectInitDescriptor.java

```diff
@@ -239,15 +239,11 @@ private void addTestFramework(BuildInitTestFramework testFramework, BuildScriptB
             case JUNIT_JUPITER:
                 buildScriptBuilder
                     .testImplementationDependency(
-                        "Use JUnit Jupiter API for testing.",
-                        "org.junit.jupiter:junit-jupiter-api:" + libraryVersionProvider.getVersion("junit-jupiter")
-                    ).testRuntimeOnlyDependency(
-                    "Use JUnit Jupiter Engine for testing.",
-                    "org.junit.jupiter:junit-jupiter-engine"
-                ).taskMethodInvocation(
-                    "Use junit platform for unit tests.",
-                    "test", "Test", "useJUnitPlatform"
-                );
+                        "Use JUnit Jupiter for testing.",
+                        "org.junit.jupiter:junit-jupiter:" + libraryVersionProvider.getVersion("junit-jupiter"))
```

Comment:
Should JUnit's documentation be aligned with this recommendation?
AFAIK, the latest JUnit documentation still suggests adding `junit-jupiter-api` and `junit-jupiter-engine` dependencies (see https://junit.org/junit5/docs/5.8.0-M1/user-guide/#running-tests-build-gradle-engines-configure )

Reply:
Interesting. I also did not know about this. Since when does this work without explicitly depending on an engine?

We should also adjust all samples and snippets in the Gradle docs that are maintained manually. Would you care to do a follow up for that @marcphilipp? doing a string search for `junit-jupiter-engine` in `subprojects/docs/src` should reveal the places.

Reply:
It was [introduced in 5.4.0](https://junit.org/junit5/docs/5.4.0/release-notes/index.html). `junit-jupiter` is basically an empty JAR (except for a module descriptor) that pulls in `junit-jupiter-api` and `junit-jupiter-params` as transitive compile dependencies and `junit-jupiter-engine` as runtime dependency.

Reply:
> Should JUnit's documentation be aligned with this recommendation?

Good catch! Done in https://github.com/junit-team/junit5/pull/2595.

Reply:
> We should also adjust all samples and snippets in the Gradle docs that are maintained manually. Would you care to do a follow up for that @marcphilipp?

Will do.